src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -262,16 +260,22 @@ class BreadcrumbsContainer extends React.Component<Props, State> {
       })
     );
 
-    this.setState({
-      searchTerm,
-      filteredBreadcrumbs,
-    });
+    this.setState(
+      {
+        searchTerm,
+        filteredBreadcrumbs,
+      },
+      this.collapsedCrumbs
```

Comment:
Why does this need to be called as a setState callback? This is guaranteed to cause at least 2 renders. Can we update state without the callback so that React can batch the updates as necessary?